remill/BC/Compat/DataLayout.h

```diff
@@ -0,0 +1,11 @@
+#include <llvm/IR/DataLayout.h>
+
```

Comment:
For this file, it would be easier to have it more like..

```c++
#if ....
old version of functions here
#else 
new version of functions here
#endif 
```

I say this because `ToSize` needs to return an `llvm::TypeSize` in LLVM 10, and a `uint64_t` in versions less than 10.

Also, lets rename `ToSize` to be `BitsToTypeSize` or something.

Reply:
done